content/applications/inventory_and_mrp/maintenance/maintenance_setup.rst

```diff
@@ -0,0 +1,239 @@
+   on an existing maintenance request. From the resulting popover, locate the :guilabel:`Technician`
+   field. The name listed in the field is the team member, and is the user responsible for that
+   particular request.
+
+   .. image:: maintenance_setup/maintenance-setup-popover-technician.png
+      :align: center
+      :alt: Maintenance request popover with Technician field shown.
+
+   At the far-right side of the page is a sidebar column, containing a minimized calendar set to
+   today's date, and a drop-down :guilabel:`Technician` list, displaying all the technicians (or
+   team members) with requests currently open.
+
+Equipment
+=========
+
+In Odoo *Maintenance*, *equipment* refers to machines and tools used internally in warehouse *work
+centers*. Equipment can include technology such as computers or tablets, power tools, machines used
+for manufacturing, and more.
+
+Equipment categories
+--------------------
+
+Each piece of equipment belongs to an *equipment category*. Before adding new equipment, make sure
+that a fitting equipment category is created.
+
+To create a new equipment category, navigate to :menuselection:`Maintenance app --> Configuration
+--> Equipment Categories`, and click :guilabel:`New`. Doing so opens a blank equipment category
+form.
+
+.. image:: maintenance_setup/maintenance-setup-category-form.png
+   :align: center
+   :alt: Equipment category form with various information filled out.
+
+On the blank form, assign a name in the :guilabel:`Category Name` field.
+
+In the :guilabel:`Responsible` field, assign a user to be responsible for the equipment in this
+category, if necessary. If in a multi-company environment, click the drop-down menu in the
+:guilabel:`Company` field, and select the company in the database to whom the equipment in this
+category belongs.
+
+In the :guilabel:`Email Alias` field, assign an email alias to this category, if necessary.
+
+In the :guilabel:`Comments` field, type any comments or notes for internal users to reference in
+relation to this category, if necessary.
+
+Once ready, click :guilabel:`Save` to save changes.
+
+.. note::
+   Once a new equipment category is created, all equipment belonging to that category, as well as
+   any past or currently open maintenance requests, are available from the equipment category form.
+
+   Navigate to :menuselection:`Maintenance app --> Configuration --> Equipment Categories`, and
+   select a category to view. Locate the :guilabel:`Equipment` and :guilabel:`Maintenance` smart
+   buttons at the top of the form.
+
+   .. image:: maintenance_setup/maintenance-setup-smart-buttons.png
+      :align: center
+      :alt: Equipment and Maintenance smart buttons on equipment category form.
+
+   Click the :guilabel:`Equipment` smart button to view all equipment belonging to this category.
+   Click the :guilabel:`Maintenance` smart button to view any past or currently open maintenance
+   requests.
+
+Machines & tools
+----------------
+
```

Comment:
I think the introductory sentence for the equipment section touched on machines & tools, and since this is a subsection I believe it's clear from the way it was introduced before. But, if I receive this suggestion again I will add something to clarify